src/redux/books/books.js

```diff
@@ -1,8 +1,35 @@
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const GET_BOOK_LIST = 'bookStore/books/GET_BOOK_LIST';
 
 const initialState = [];
 
+const addBookToAPI = async (book) => {
+  await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/eTfxKsR9wf3ubgm9W4xT/books', {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: book.item_id,
+      title: book.title,
+      category: book.category,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+    .then();
+};
+
+const deleteBookfromAPI = async (book) => {
+  const id = book.item_id;
+  await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/eTfxKsR9wf3ubgm9W4xT/books/${id}`, {
+    method: 'DELETE',
+    body: JSON.stringify({
+      id,
+    }),
+  })
+    .then();
+};
```

Comment:
**Optional Changes**
- [ ] Nice try using the `async await` syntactic sugar to wrap your asynchronous code. However, you're mixing it with the `then` keyword. Let us remove the `then` keyword and just use the pure `async await`.